appserver/admingui/cluster/src/main/resources/org/glassfish/cluster/admingui/Strings.properties

```
@@ -216,6 +216,7 @@ standaloneInstancePropertiesPageTitleHelp=
 standalone.ApplicationsTitle=Applications
 standalone.ApplicationsTitleHelp=Deploy, undeploy, enable, disable, and remove selected applications associated with the instance. Removing the application only removes the application from this instance. The application is not undeployed.
 standalone.InstanceName=Instance Name:
+standalone.asadminListenerPortWarning=Warning: ASADMIN_LISTENER_PORT can not be changed dynamically. You should stop the running instance before saving, otherwise DAS won't be able to stop the running instance and it may need to be forcibly stopped.
```

This message will always be displayed on page load even if the property isn't changed. I would change the text to Note: instead of Warning: so that users aren't worried that something is wrong.
```suggestion
standalone.asadminListenerPortWarning=Note: ASADMIN_LISTENER_PORT can not be changed dynamically. You should stop the running instance before saving, otherwise DAS won't be able to stop the running instance and it may need to be forcibly stopped.
```